keys.json

```
@@ -38,5 +38,6 @@
     "crex24":        { "skip": true },
     "btcex":         { "skip": true },
     "wazirx":        { "skipWs": true },
-    "deribit":       { "skipWs": true }
+    "deribit":       { "skipWs": true },
+    "theocean":      { "skip": true }
```

we dont need theocaen at all.
```suggestion
    "deribit":       { "skipWs": true }
```

---

@ttodua yeah my bad I saw a `theocean` file in the file dir and didn't notice it was an example, but it's not critical we can remove it afterwards